lib/fluent/log/console_adapter.rb

```diff
@@ -0,0 +1,53 @@
+#
+# Fluentd
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+require 'console/serialized/logger'
+
+module Fluent
+  class Log
+    class ConsoleAdapter < Console::Terminal::Logger
+      def self.wrap(logger)
+        _, level = Console::Logger::LEVELS.find { |key, value|
+          if logger.level <= 0
+            key == :debug
+          else
+            value == logger.level - 1
+          end
+        }
+        Console::Logger.new(ConsoleAdapter.new(logger), level: level)
+      end
+
+      def initialize(logger)
+        @logger = logger
+        super(StringIO.new, verbose: false)
```

Comment:
I'm still considering `verbose` should be enabled or not.

Reply:
When `verbose=true`, following items are added as a prefix & suffix of subject:

* Severity
* Object ID
* pid
* Time

Severity and Time are added by our logger, so they are just redundant.
`pid` should be same with the worker, so it's also redundant.
Object ID seems too verbose in usual cases.

So I think `verbose=false` seems reasonable for us.